cloud

```diff
@@ -702,23 +717,27 @@ function scheduler_job_withdraw() {
 }
 
 # Deploy to scheduler
-# Args: project prefix trigger_topic
+# Args: project prefix load_queue_trigger_topic pick_notifications_trigger_topic
 function scheduler_deploy() {
     declare -r project="$1"; shift
     declare -r prefix="$1"; shift
-    declare -r trigger_topic="$1"; shift
+    declare -r load_queue_trigger_topic="$1"; shift
+    declare -r pick_notifications_trigger_topic="$1"; shift
     # Deploy the jobs
-    scheduler_job_pubsub_deploy "$project" "${prefix}trigger" \
-                                "$trigger_topic" '* * * * *' '{}'
+    scheduler_job_pubsub_deploy "$project" "${prefix}load_queue_trigger" \
+                                "$load_queue_trigger_topic" '* * * * *' '{}'
+    scheduler_job_pubsub_deploy "$project" "${prefix}pick_notifications_trigger" \
+                            "$pick_notifications_trigger_topic" '30 * * * *' '{}'
 }
 
 # Withdraw from the scheduler
 # Args: project prefix
 function scheduler_withdraw() {
     declare -r project="$1"; shift
     declare -r prefix="$1"; shift
-    declare -r trigger_topic="$1"; shift
-    scheduler_job_withdraw "$project" "${prefix}trigger" "$trigger_topic"
+    declare -r load_queue_trigger_topic="$1"; shift
+    scheduler_job_withdraw "$project" "${prefix}trigger"\
+                            "$load_queue_trigger_topic"
```

Comment:
Could you move this line one character to the left so it aligns with the first argument?